mod/invlfire.mod

```diff
@@ -207,24 +210,55 @@ static void bbcore_write(double* x, int* d, int* xx, int *offset, _threadargspro
         fprintf(stderr, "InvlFire: Random123 generator is required\n");
         assert(0);
       }
+      nrn_random123_getseq(*pv, di+3, &which);
+      di[4] = (int)which;
     }else{
 #else
     {
 #endif
       nrnran123_State** pv = (nrnran123_State**)(&_p_r);
       nrnran123_getids3(*pv, di, di+1, di+2);
+      nrnran123_getseq(*pv, di+3, &which);
+int z = 0;
+#if NRNBBCORE
+z = 1;
+#endif
```

Comment:
@nrnhines : Just curious - is this used for something?

Reply:
It was sometimes useful in printf statements to indicate whether the print was coming from neuron or coreneuron. Since the debugging printf has served its purpose and was removed, I suppose the z fragment should be removed as well.